pulsar-sql/pom.xml

```diff
@@ -37,6 +37,13 @@
         <module>presto-distribution</module>
     </modules>
 
+    <properties>
+        <!-- keep using okhttp3 3.x for Presto -->
+        <okhttp3.version>3.14.9</okhttp3.version>
+        <!-- use okio version that matches the okhttp3 version -->
+        <okio.version>1.17.2</okio.version>
```

Comment:
Why is a different version of okio used in this pom from the min pom?

Reply:
this is the pom of presto-sql which is going to be moved to the presto repository (see [PIP-62](https://github.com/apache/pulsar/wiki/PIP-62%3A-Move-connectors%2C-adapters-and-Pulsar-Presto-to-separate-repositories))

I don't want to introduce potential problems here, so I left the current version for okhttp3 and okio (for okio: I forced the same one used by okhttp3)